build.sbt

```diff
@@ -69,7 +69,7 @@ lazy val root = project.in(file(".")).settings(buildSettings).aggregate(core, ki
 
 lazy val core = project
   .settings(moduleName := "snowplow-stream-collector-core")
-  .settings(buildSettings)
+  .settings(buildSettings ++ BuildSettings.sbtAssemblySettings)
```

Comment:
We've added assembly, but where's the logic of actual publishing? Just FYI we shoudn't publish jars on S3 anymore - GH assets should be preferred https://github.com/snowplow-incubator/snowplow-bigquery-loader/issues/135#issuecomment-741006011